config/locales/views/admin/en.products.yml

```diff
@@ -34,6 +34,10 @@ en:
         index:
           description: Services are provided by the !facility_downcase! on behalf of the user. Forms for collection of additional information may be attached to the service.  Results files may be returned to the user.
 
+      timed_services:
+        index:
+          description: Timed services are provided by the !facility_downcase! on behalf of the user; billing is based on time used.
```

Comment:
The text in this spot for instruments doesn't specify units--the pricing units are specified in the form (for both Instruments and TimedServices). I think it would be kind of confusing to include the units here, honestly--because we charge by the minute, but they set the rates by hour. It's much clearer in the form.